contracts_api/resources/__init__.py

```diff
@@ -0,0 +1,23 @@
+from sanic.response import json, text
+from sanic.views import HTTPMethodView
+
+
+class SmokeResources(HTTPMethodView):
+
+    async def smoke(self, *args, **kwargs):
```

Comment:
smoke endpoint means that it's just a default GET endpoint, but it returns simple response. Remove async def smoke